jplag.frontend.R/src/main/java/de/jplag/R/RToken.java

```diff
@@ -0,0 +1,47 @@
+package de.jplag.R;
+
+/**
+ * This class represents the occurrence of an R Token in the source code. Based on an R frontend for JPlag v2.15 by Olmo
+ * Kramer, see their <a href="https://github.com/CodeGra-de/jplag/tree/master/jplag.frontend.R">JPlag fork</a>.
+ * @author Robin Maisch
```

Comment:
What is our stance on Author annotations @tsaglam?

Reply:
Currently, we have no rules that we enforce. Historically, some classes have author tags and many don't. I use it for new classes or classes I rewrite from scratch. However, we cannot really enforce contributors to put their full names in the code, as many of them do not have their names on GitHub. So for now, I think it is fine to add the tags if you want to. I find them useful to see which person and thus in which context a class was written. I would not add an author tag to existing classes that I just edited.

Reply:
You are right, I don't remember why I put that there. Removed.